### PR TITLE
Allow to import files using soffice

### PR DESCRIPTION
**Pull request by ldidry**
Currently, you have to use Abiword to convert odt, pdf and doc files to be able to import those files.
This add soffice as convertor too.

### CODE REVIEW

**Comment by ldidry** [comment]
ping @lpagliari ?

**Comment by lpagliari** [comment]
Sorry, I’m in the middle of my vacations, will only be able to take a look
on it in April

**Comment by ldidry** [comment]
Oh, sorry.

I was just that you answered on #3338 really quickly, and this PR is quite on the same scope than #3338, I wondered why you didn't reviewed it.

Enjoy your vacations 😊

**Comment by ldidry** [comment]
ping @lpagliari ? (if you have time)

**Comment by ldidry** [comment]
Hello, can someone look this PR please?

File: src/node/handler/ImportHandler.js [hunk 1]
```diff
@@ -34,9 +34,18 @@ var ERR = require("async-stacktrace")
   , log4js = require("log4js")
   , hooks = require("ep_etherpad-lite/static/js/pluginfw/hooks.js");
 
+var convertor = null;
+var exportExtension = "htm";
```
[the remark this answers is not in the document]
Comment (ldidry) [reply src/node/handler/ImportHandler.js]: ".htm" is used with Abiword for a very long time (like more than 6 years), I don't know if it's required for Abiword, but I know that LOffice does not like it and want ".html" to work properly.

File: src/node/handler/ImportHandler.js [hunk 2]
```diff
@@ -34,9 +34,18 @@ var ERR = require("async-stacktrace")
   , log4js = require("log4js")
   , hooks = require("ep_etherpad-lite/static/js/pluginfw/hooks.js");
 
+var convertor = null;
```
[the remark this answers is not in the document]
Comment (muxator) [reply src/node/handler/ImportHandler.js]: Thanks for the advice

File: src/node/handler/ImportHandler.js [hunk 3]
```diff
@@ -34,9 +34,18 @@ var ERR = require("async-stacktrace")
...
+var exportExtension = "htm";
+
 //load abiword only if its enabled
 if(settings.abiword != null)
-  var abiword = require("../utils/Abiword");
+  convertor = require("../utils/Abiword");
```
[the remark this answers is not in the document]
Comment (ldidry) [reply src/node/handler/ImportHandler.js]: See #3507 for the new PR, sorry for the mess.

**Pull request closed without merging**